.eslintrc.json

```diff
@@ -8,7 +8,6 @@
 
     "parserOptions": {
         "ecmaFeatures": {
-            "experimentalObjectRestSpread": true,
```

Comment:
Like nothing in TS works anymore, or it just gives some warning on that line in intellij?